pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -69,7 +69,7 @@ func (podDisruptionBudgetStrategy) PrepareForUpdate(ctx context.Context, obj, ol
 // Validate validates a new PodDisruptionBudget.
 func (podDisruptionBudgetStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
 	podDisruptionBudget := obj.(*policy.PodDisruptionBudget)
-	return validation.ValidatePodDisruptionBudget(podDisruptionBudget)
+	return validation.ValidatePodDisruptionBudgetStrict(podDisruptionBudget)
```

Comment:
rather than name this `Strict` (which is unclear what it is being strict about), create a PodDisruptionBudgetValidationOptions type containing an `AllowInvalidSelector bool` field and pass it into ValidatePodDisruptionBudget().

From this callsite, pass `PodDisruptionBudgetValidationOptions{AllowInvalidSelector: false}`

From ValidateUpdate, set AllowInvalidSelector to `true` if the old PDB has an invalid selector.